modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language_ar.properties

```diff
@@ -16189,7 +16189,7 @@ there-are-no-site-templates=لا توجد قوالب للموقع.
 there-are-no-sources=لا يوجد مصادر.
 there-are-no-stores-configured=لا توجد متاجر تم تكوينها.
 there-are-no-structures=لا توجد بنيات.
-there-are-no-style-books=لا يوجد أي سجلات أنماط.
+there-are-no-style-books=There are no style books. (Automatic Copy)
```

Comment:
Hello @SelenaAungst, it looks like it did not translate correctly. Can you try rebuilding this and see if it will translate correctly?